platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/transaction/Transaction.java

```diff
@@ -46,7 +45,7 @@ public sealed interface Transaction permits ConsensusTransaction {
      * @return the application transaction Bytes or null if the transaction is a system transaction
```

Comment:
seems this javadoc is innacurate, it says it will return null, but it wont

Reply:
fixed